hicommonwealth/commonwealth
# Fix view snapshot page

## Link to Issue
Closes: #3301 

## Description of Changes
- Refactors snapshot proposal page from Mithril to React, which seems to result in more consistent rendering
- Fixes className syntax bug
- Fixes snapshot proposal list item URL to navigate to the correct page

## Test Plan 1 (proposal list view flow)
- Navigate to http://localhost:8080/dydx/snapshot/dydxgov.eth
- If no active proposals exist, go to the Ended tab
- Click a proposal list item– confirm that it navigates to the proposal view page
- Confirm that the proposal view page renders all components correctly

## Test Plan 2 (fresh page load flow)
- Navigate directly to http://localhost:8080/dydx/snapshot/dydxgov.eth/0x31f58b0bef1cbc336fd0f65a52e8add8943e1448afe7fc32da85744501020148
- Confirm that the proposal view page renders all components correctly

## Deployment Plan

N/A

## Other Considerations

- Need help testing on spaces other than dydx. I've tested on dydx, but I'm unclear on how navigate to other spaces and view their snapshots.
